### ISSUE TITLE
Manage users. Missing translation for folder selection summary

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#160]
### Description of issue

We are missing 3 translations:

1. Summary when no folders are selected
2. Summary when some folders are selected
3. The "Done" button, including its visually hidden text "selecting folders"

> <img width="496" alt="Capture d’écran, le 2021-10-27 à 14 23 58" src="https://user-images.githubusercontent.com/5230720/139124760-45ebe8e1-08db-4484-9611-b95fe9d70912.png">

> <img width="553" alt="Capture d’écran, le 2021-10-27 à 14 23 51" src="https://user-images.githubusercontent.com/5230720/139124729-60be9a7c-fb23-4430-81c0-67da03d63e56.png">

> <img width="568" alt="Capture d’écran, le 2021-10-27 à 14 24 03" src="https://user-images.githubusercontent.com/5230720/139124784-d076bbe3-f66a-4da3-a853-01e3210295a6.png">

> <img width="385" alt="Capture d’écran, le 2021-10-27 à 14 25 19" src="https://user-images.githubusercontent.com/5230720/139124814-71e5caac-9bcd-44d8-9286-33d77895c69e.png">

### Potential fix

- Find where these strings are set. Like here

https://github.com/cds-snc/notification-admin/blob/082075c5e7bf0a7d22aaeb899a9d0e6df276c002/app/assets/javascripts/collapsibleCheckboxes.js#L30-L32
